backend/user/src/services/auth/types.ts

```diff
@@ -1,4 +1,68 @@
+import type { IServiceResponse } from '@/types';
+
+//=============================================================================
+// /auth/login
+//=============================================================================
+
 export type ILoginPayload = {
   username: string;
   password: string;
 };
+
+export type ILoginResponse = IServiceResponse<{
+  cookie: string;
```

Comment:
JWT Token is sent as HttpOnly cookie. No accessible cookie is sent to the frontend. Suggest to remove `cookie` field

Reply:
I might want to add the distinction for this case that this service returns the cookie to the controller, which then sets it on the response hence it is included here